presto-hive-common/src/main/java/com/facebook/presto/hive/BaseHiveClientConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+
+public class BaseHiveClientConfig
```

Comment:
Why introduce this? I don't see it's used. I guess it's because SubfieldExtractor was moved to presto-hive-common, But I don't see HIveClientConfig extending BaseHiveClientConfig. In HIveClientConfig, there is another isRangeFiltersOnSubscriptsEnabled defined.

Reply:
Yes, that's right these are required because of movement of SubfieldExtractor into presto-hive-common. I think rather than extending HiveClientConfig, we can bind BaseHiveClientConfig in HiveClientModule. Let me know what you think about this?

>  In HIveClientConfig, there is another isRangeFiltersOnSubscriptsEnabled defined.
Yes, my bad, forgot to remove this from HiveClientConfig.